CHANGELOG.md

```diff
@@ -5,6 +5,23 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 ## Unreleased
+
+## [v16.0.0] 2023-05-09
+
+### MIGRATION notes
+
+#### PI release version
+
+When updating directly to v16 from prior releases older that V15, please make sure to
+read through all prior release notes.
+
+Notable migration concerns since the last PI release version (11.1.x):
```

Comment:
Wasn't the last PI release version v 14?

Reply:
Did we maybe mean "minimum supported version"?